### PR TITLE
nimble/btshell: Add support for toggling reporting

### PR DESCRIPTION
**Pull request by szymon-czapracki**
Allows to enable periodic advertising sync reporting.

### CODE REVIEW

**Comment by szymon-czapracki** [comment]
@sjanc FYI

File: apps/btshell/src/cmd.c [hunk 1]
```diff
@@ -4550,6 +4595,13 @@ static const struct shell_cmd btshell_commands[] = {
         .sc_cmd_func = cmd_sync_transfer_receive,
 #if MYNEWT_VAL(SHELL_CMD_HELP)
         .help = &sync_transfer_receive_help,
+#endif
+    },
+    {
+       .sc_cmd = "sync_reporting",
```
Comment (andrzej-kaczmarek) [inline apps/btshell/src/cmd.c]: should be `sync-reporting` (same as other sync commands)

File: apps/btshell/src/cmd.c [hunk 2]
```diff
@@ -3861,6 +3894,18 @@ static const struct shell_cmd_help sync_transfer_help = {
...
 };
+
+static const struct shell_param sync_reporting_params[] = {
+    {"sync_handle", "sync handle, usage: =[UINT16], default: 0"},
+    {"toggle_reporting", "toggle reporting, usage: =[0-1], default: 0"},
+    {NULL, NULL}
+};
+
+static const struct shell_cmd_help sync_reporting_help = {
+   .summary = "start periodic advertising sync reporting",
```
Comment (andrzej-kaczmarek) [inline apps/btshell/src/cmd.c]: better describe it as "configure" instead of "start"

File: apps/btshell/src/cmd.c [hunk 3]
```diff
@@ -3861,6 +3894,18 @@ static const struct shell_cmd_help sync_transfer_help = {
     .usage = NULL,
     .params = sync_transfer_params,
 };
+
+static const struct shell_param sync_reporting_params[] = {
+    {"sync_handle", "sync handle, usage: =[UINT16], default: 0"},
+    {"toggle_reporting", "toggle reporting, usage: =[0-1], default: 0"},
```
Comment (andrzej-kaczmarek) [inline apps/btshell/src/cmd.c]: `enabled`

**Pull request merged**